### DISCUSSION TITLE
OpenPDF PDF/A conformance not given through missing of attribute xml:lang="x-default" dc:title -> rdf:li

### DISCUSSION THREAD

**Discussion by speckyspooky** [LibrePDF/OpenPDF#969] [Q&A]
Hello There,

I created different PDF/A1-A & -B documents with openPdf and checked the quality.
The main problem is that the dc:title will be noted as uncorrect due to the missing language option which is an attribute value.

The required versio of the XMP metadata is:
<dc:title><rdf:Alt><rdf:li xml:lang="x-default">Titel-Subtitel</rdf:li></rdf:Alt></dc:title>

The given version of openPDF is:
<dc:title><rdf:Alt><rdf:li>Titel-Subtitel</rdf:li></rdf:Alt></dc:title>

I tested with the page:
https://www.pdfforge.org/online/de/pdfa-validieren

Is there an option to activate the language attribute so that the pdf is a valid PDF/A document?

**Comment by andreasrosdalw** [comment]
Please see #980 - we would appreciate someone contributing PDF/A support in OpenPDF.

**Comment by asturio** [comment]
I would be interesting to know what is the current status. Did OpenPDF solved this already? If not, pull requests are still welcome.
